README.md

```diff
@@ -5,46 +5,196 @@
+    * [core](https://github.com/openvinotoolkit/openvino/tree/update_readme/src/core) - provides the base API for model representation and modification.
+    * [inference](https://github.com/openvinotoolkit/openvino/tree/update_readme/src/inference) - provides an API to infer models on device.
+    * [transformations](https://github.com/openvinotoolkit/openvino/tree/master/src/common/transformations) - contains the set of common transformations which are used in OpenVINO plugins.
+    * [low precision transformations](https://github.com/openvinotoolkit/openvino/tree/master/src/common/low_precision_transformations) - contains the set of transformations which are used in low precision models
+    * [bindings](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings) - contains all awailable OpenVINO bindings which are maintained by OpenVINO team.
+        * [c](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/c) - provides C API for OpenVINO™ Runtime
+        * [python](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/python) - Python API for OpenVINO™ Runtime
+* [Plugins](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins) - contains OpenVINO plugins which are maintained in open-source by OpenVINO team. For more information please taje a look to the [list of supported devices](#supported-hardware-matrix).
+* [Frontends](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends) - contains available OpenVINO frontends which allow to read model from native framework format.
+    * [IR](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/ir) allows to read model from OpenVINO IR format.
+    * [ONNX](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx) allows to read native ONNX models.
+    * [Paddle Paddle](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/paddle) allows to read native Paddle Paddle models.
+    * [Tensorflow](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow) allows to read native Tensorflow models.
+* [Model Optimizer] - is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
+* [Post-Training Optimization Tool] - is designed to accelerate the inference of deep learning models by applying special methods without model retraining or fine-tuning, for example, post-training 8-bit quantization. 
+* [Samples] - applications on C, C++ and Python languages which shows basic use cases of OpenVINO usages.
+
+## Supported Hardware matrix
+
+The OpenVINO™ Runtime can infer models on different hardware devices. This section provides the list of supported devices.
+
+<table>
```

Comment:
Width of the `Short Description` section is quite tight, and it's hard to read this section. 

As a quick fix, `Location` section could be removed, and the URL link in the location could be inserted into Library item. For example, the first row has `Library` `openvino_intel_cpu_plugin` and `Location` [openvino/src/plugins/intel_cpu](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu). Instead, `openvino/src/plugins/intel_cpu` could be removed and the link could be inserted into [openvino_intel_cpu_plugin](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu) in `Library` column. This way some space would be gained, and the description section would be a bit larger, ultimately would improve the readability.

Another alternative would be to place `Description` under `Plugin`, `Library` and `Location` as a new cell.

Reply:
Thanks fixed